pandas/core/generic.py

```diff
@@ -7660,6 +7618,204 @@ def nanptp(values, axis=0, skipna=True):
             ``numpy.ndarray`` method ``ptp``.""",
             nanptp)
 
+    _shared_docs['transform'] = ("""
+    Call function on a Series resp. DataFrame
+    producing a like-indexed Series resp.
+    DataFrame with the transformed values.
+
+    .. versionadded:: 0.20.0
+
+    Parameters
+    ----------
+    func : Function, dictionary, or list of functions
+
+        Function: Function to apply to the DataFrame/Series.
```

Comment:
no blank line here

Reply:
blank line 7631 removed.   
(But I think the rendering is not as nice)